frontend/src/buttons/LaunchButton/LaunchButton.tsx

```diff
@@ -42,7 +44,10 @@ export const LaunchButton: React.FC<Props> = ({ connection, service }) => {
         ...connection,
         username: username.toString(),
       })
-    window.open(app.launch({ ...connection, username }))
+    const launchApp = app.launch({ ...connection, username })
+    console.log({app, launchApp})
+    isWindows() && app.title === 'SSH' ?
```

Comment:
Can we move this logic into the `/shared/applications.ts` file?

Please add a new method on the Application class that returns a boolean if it should launch to the browser.

So on the base class the method could return by default `true`.

Then the instance of 'SSH' should have an override where it checks for isWindows().

Let me know if that all makes sense... I'm happy to jump on a call!  :)